### DISCUSSION TITLE
Release notes for 5.1.0 mentions upgrade to asp.net core 1.1.1 but not in source?

### DISCUSSION THREAD

**Discussion by dotnetshadow** [Dotnet-Boxed/Templates#147] [Q&A]
Hi there,

I noticed you mention in the release notes for 5.1.0 you upgraded to asp.net core 1.1.1 but for some reason I don't see it in project.json unless the change got reverted?

**Comment by RehanSaeed** [comment]
Have not released yet. Will do so this weekend.

**Comment by dotnetshadow** [comment]
Awesome thanks
